src/main/java/org/folio/circulation/rules/CirculationRulePolicyIdEntity.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.rules;
+
+import java.util.List;
+
+public class CirculationRulePolicyIdEntity {
+
+  private final String policyId;
+  private final List<String> ruleConditions;
+
+  public CirculationRulePolicyIdEntity(String policyId, List<String> ruleConditions) {
+    this.policyId = policyId;
+    this.ruleConditions = ruleConditions;
+  }
+
+  public String getPolicyId() {
+    return policyId;
+  }
+
+  public List<String> getRuleConditions() {
+    return ruleConditions;
+  }
+
+  @Override
+  public String toString() {
```

Comment:
Why has the `toString` method been overridden?

The implementation seems to suggest manually constructed JSON, is that the case?

Reply:
removed